model_to_project_task/action.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+# © 2015 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, _
```

Comment:
Did I misunderstood this part?
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports
> from openerp import api, fields, models  # alphabetically ordered